dex-load/src/main/scala/com/wavesplatform/dex/load/RequestTag.scala

```diff
@@ -0,0 +1,62 @@
+package com.wavesplatform.dex.load
+
+import java.io.PrintWriter
+
+import com.google.common.net.HttpHeaders
+import com.wavesplatform.dex.load.RequestType.RequestType
+import com.wavesplatform.dex.load.RequestTag.RequestTag
+import com.wavesplatform.dex.load.utils._
+import com.wavesplatform.wavesj.ApiJson
+
+object RequestTag extends Enumeration {
+  type RequestTag = Value
+  val RESERVED_BALANCE      = Value("RESERVED_BALANCE")
+  val TRADABLE_BALANCE      = Value("TRADABLE_BALANCE")
+  val ORDER_BOOK_BY_PAIR = Value("ORDER_HISTORY_BY_PAIR")
+  val ORDER_HISTORY_BY_ADDRESS  = Value("ORDER_HISTORY_BY_ACC")
+  val ORDER_STATUS          = Value("ORDER_STATUS")
+  val CANCEL                = Value("CANCEL")
+  val PLACE                 = Value("PLACE")
```

Comment:
As @vsuharnikov once said, its enough to simply write 
```
val RESERVED_BALANCE, TRADABLE_BALANCE, ORDER_BOOK_BY_PAIR, ORDER_HISTORY_BY_ADDRESS, ORDER_STATUS, CANCEL, PLACE = Value
```

Reply:
done